app/models/knight.rb

```diff
@@ -1,2 +1,17 @@
 class Knight < ChessPiece
+  XY_MOVE_FACTOR = 1
+  ALT_XY_MOVE_FACTOR = 2
```

Comment:
Probably...I struggled myself. Basically, I am trying to express that both x and y can either be 1 or 2 - so I modeled this concept of the `XY_MOVE_FACTOR` as being 1 and the alternative as being 2.

It sucks and I am totally open to other concepts... :)